src/configuration.ts

```diff
@@ -63,7 +64,10 @@ export interface IConfigurationLoadResult {
     results?: IConfigurationFile;
 }
 
-export const CONFIG_FILENAME = "tslint.json";
```

Comment:
this could actually break some users of our API:

```ts
import * as Lint from "tslint";
Lint.Configuration.CONFIG_FILENAME; // no longer valid
```

I doubt this is used in real world code, so I don't know if this is worth fixing.